packages/askui-nodejs/src/execution/dsl.ts

```diff
@@ -12,6 +12,8 @@ export enum Separators {
   STRING = '<|string|>',
 }
 
+export type Relations = 'nearestTo' | 'leftOf' | 'above' | 'rightOf' | 'below' | 'contains';
```

Comment:
You are right, the whole `dsl.ts` file is generated through the askui-inference. @programminx-askui or @mlikasam-askui can help you out with interating it there. Alternatively, you could also simply move this to another file, e.g., `ui-control-client.ts` or a new file.